src/import/ensureConfiguration.ts

```diff
@@ -0,0 +1,596 @@
+  const params: { [key: string]: boolean } = { 'show-inherited-invariants': false }; // backwards-compatible to IG Exporter
+  if (igIni['usage-stats-opt-out']) {
+    params['usage-stats-opt-out'] = igIni['usage-stats-opt-out'];
+  }
+  contents.addPair(
+    'parameters',
+    params,
+    'NOTE: The parameters property represents IG.definition.parameter. Rather than a list of code/value ' +
+      'pairs (as in the ImplementationGuide resource), the code is the YAML key. If a parameter allows repeating ' +
+      'values, the value in the YAML should be a sequence/array. For a partial list of allowed parameters see: ' +
+      'https://confluence.hl7.org/display/FHIR/Implementation+Guide+Parameters'
+  );
+  // copyrightYear
+  contents.addPair(
+    'copyrightYear',
+    igIni.copyrightyear ?? DEFAULT_IG_INI.copyrightyear,
+    'NOTE: The following two lines correspond to items that are required for implementation guides, and used to be ' +
+      'in ig.ini but were moved to IG.definition.parameter. Given that they are required, they are specified as ' +
+      'top-level properties in SUSHI configuration, but SUSHI will convert them to ImplementationGuide parameters using ' +
+      'the formal parameter names: copyrightyear and releaselabel.'
+  );
+  // releaseLabel
+  contents.addPair('releaseLabel', igIni.ballotstatus ?? DEFAULT_IG_INI.ballotstatus);
+  // template
+  if (igIni !== DEFAULT_IG_INI) {
+    contents.addCommentedOutPair(
+      'template',
+      igIni.template ?? DEFAULT_IG_INI.template,
+      'NOTE: If template does not exist (or is commented out), then SUSHI will not generate an ig.ini file; but if ' +
+        'ig-data/ig.ini exists, SUSHI will copy that ig.ini to the appropriate destination path. Uncomment template ' +
+        'to manage the ig.ini contents in this configuration file instead. Currently ig.ini supports only two ' +
+        'properties: ig (the path to the ImplementationGuide resource) and template. For more on template-based ' +
+        'publishing, see: https://build.fhir.org/ig/FHIR/ig-guidance/index.html#technical-details.'
+    );
+  } else {
+    contents.addPair(
+      'template',
+      igIni.template ?? DEFAULT_IG_INI.template,
+      'NOTE: The template property is used to generate an ig.ini file. Currently ig.ini supports only two ' +
+        'properties: ig (the path to the ImplementationGuide resource) and template. To manage your own ig.ini ' +
+        'file, delete or comment out template and provide an ig.ini at ig-data/ig.ini. For more on template-based ' +
+        'publishing, see: https://build.fhir.org/ig/FHIR/ig-guidance/index.html#technical-details.'
+    );
+  }
+  // menu
+  const menuPath = path.join(root, 'ig-data', 'input', 'includes', 'menu.xml');
+  if (fs.existsSync(menuPath)) {
+    const menuXML = fs.readFileSync(menuPath, 'utf8');
+    const menu = getMenuObjectFromMenuXML(menuXML);
+    contents.addCommentedOutPair(
+      'menu',
+      menu ?? DEFAULT_MENU,
```

Comment:
If I understand this correctly, then if they have a menu at `ig-data/input/includes/menu.xml`, we will try to read that menu in, so that we can add a commented out property showing how the `menu` property would look if used in the config. But it is very possible that we can't read in their menu, so we show the DEFAULT_MENU instead. Do we possibly want to put a disclaimer here to indicate that this menu does not necessarily match their menu.xml and that they need to change menu as necessary? I only ask this in the case of menu since it seems like you feel its possible if not likely that we can't programmatically convert their menu easily/reliably.

Reply:
I was thinking that the menu format is so simple that it would likely be pretty obvious that the default menu is different and probably should be modified.  But it probably makes sense to update the comment text to suggest they uncomment it and modify it to match their desired menu contents.  or something like that.

Reply:
Again, Mark requested minimal comments, so I couldn't really expound on this, but the comment says to uncomment and populate the menu property.  So hopefully they'll at least look at it as they uncomment it...